client/my-sites/plugins/plugins-discovery-page/index.jsx

```diff
@@ -112,15 +141,22 @@ const PluginsDiscoveryPage = ( props ) => {
 				/>
 			) }
 
-			<PaidPluginsSection { ...props } />
-			<CollectionListView category="monetization" { ...props } />
-			<EducationFooter />
-			{ ! isLoggedIn && <InPageCTASection /> }
-			<FeaturedPluginsSection
-				{ ...props }
-				pluginsByCategoryFeatured={ pluginsByCategoryFeatured }
-				isFetchingPluginsByCategoryFeatured={ isFetchingPluginsByCategoryFeatured }
-			/>
+			{ isWPBeginnerSpecial ? (
+				<FeaturedWPBeginnerSection { ...props } />
+			) : (
+				<>
+					<PaidPluginsSection { ...props } />
+					<CollectionListView category="monetization" { ...props } />
+					<EducationFooter />
+					{ ! isLoggedIn && <InPageCTASection /> }
+					<FeaturedPluginsSection
+						{ ...props }
+						pluginsByCategoryFeatured={ pluginsByCategoryFeatured }
+						isFetchingPluginsByCategoryFeatured={ isFetchingPluginsByCategoryFeatured }
+					/>
+				</>
+			) }
```

Comment:
Instead of replacing those sections can we put the partner plugins on top? And perhaps we can make the implementation more abstract in the future to work with any site created through a partner flow.

Reply:
That's a good point.
I replaced it with a `FeaturePartnerBundlePlugins` that takes in the partnership as category.